ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -244,10 +244,37 @@ tests:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-      TEST_SCENARIOS: MCO|Cluster_Infrastructure
+      TEST_SCENARIOS: Cluster_Infrastructure
     test:
     - chain: openshift-e2e-test-qe-longduration
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: aws-ipi-f28-longduration-mco-critical
+  cron: 32 22 2 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "2"
+      TEST_FILTERS: ""
```

Comment:
@sergiordlr @rioliu-rh 
if you set it empty, it means there is case which has label `ChkUpgrade, DisconnectedOnly, HyperShiftMGMT, MicroShiftOnly` and it is executed here, but should not.

so, maybe you could set it as `"~ChkUpgrade&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&"`

Reply:
I checked the above filters in MCO files. only HyperShiftMGMT cases are available, and importance is High.

Reply:
I suggest to add it in case you have such label for cases in the future.

Reply:
thanks @kuiwang02 for the comment, we can add filters like `~ChkUpgrade&;~DisconnectedOnly&;~MicroShiftOnly&` except `HyperShiftMGMT`, the cases can be executed w/o hypershift env on AWS platform

Reply:
ok.